java/src/com/ibm/streamsx/topology/internal/core/StreamImpl.java

```diff
@@ -540,6 +541,12 @@ else if (width instanceof SubmissionParameter<?>)
     public TStream<T> parallel(Supplier<Integer> width) {
         return parallel(width, TStream.Routing.ROUND_ROBIN);
     }
+    
+    @Override
+    public void setParallel(Supplier<Integer> width){
+    	output._json().addProperty(PortProperties.PARTITIONED, true);
```

Comment:
I don't think it's partitioned.

Reply:
copy-paste error.

Reply:
To be consistent with `setConsistent` it should return `this`.

http://ibmstreams.github.io/streamsx.topology/doc/releases/latest/javadoc/com/ibm/streamsx/topology/TStream.html#setConsistent-com.ibm.streamsx.topology.consistent.ConsistentRegionConfig-